packages/php/email-editor/src/Engine/PersonalizationTags/class-personalization-tags-registry.php

```diff
@@ -81,6 +81,28 @@ public function register( Personalization_Tag $tag ): void {
 		);
 	}
 
+	/**
+	 * Unregister a personalization tag by its token.
+	 *
+	 * @param string $token The token of the personalization tag to unregister.
+	 * @return Personalization_Tag|null The unregistered tag or null if not found.
+	 */
+	public function unregister( string $token ): ?Personalization_Tag {
+		$tag = $this->tags[ $token ] ?? null;
```

Comment:
Can we support accepting the token or the Personalization_Tag as arguments and unregistering it if it exists?